test/src/tester.c

```diff
@@ -1175,13 +1175,43 @@ void run_static_tests (int argc, const char * argv[]){
     assert(igs_service_exists("myService"));
     listOfStrings = igs_service_list(&nbElements);
     assert(nbElements == 1 && streq(listOfStrings[0], "myService"));
+    
+    assert(igs_service_description("myService") == NULL);
+    assert(igs_service_set_description("unknow", "myService description") == IGS_FAILURE);
+    assert(igs_service_set_description("myService", "myService description") == IGS_SUCCESS);
+    char * servicedescription = igs_service_description("myService");
+    assert(strcmp(servicedescription, "myService description") == 0);
```

Comment:
You should use `streq(a, b)` instead of `strcmp(a, b) == 0` for readability and consistency between tests